docs/cli/cmdline/cloud-drift-show.md

```diff
@@ -9,4 +9,4 @@ The `cloud drift show` command shows any drift that has occurred in the current
 
 ## Usage
 
-`terramate -C <stack-directory> cloud drift show`
+`terramate [-C <stack-directory>] cloud drift show`
```

Comment:
This is not optional as drift show does not run recursivly. it is only optional if you are in the stack.

Reply:
```
$ terramate cloud drift show
Error: No stack selected. Please enter a stack to show a potential drift.
```

Reply:
It's an optional flag, so it's correct to use `[-C <stack dir>]`. Maybe it needs additional explaining that you need either enter the directory or use `-C`.

Reply:
At the moment it is required, we can not let users run into errors. Once we finished the feature, we can fix the docs